docs/syscall_descriptions.md

```diff
@@ -148,6 +148,9 @@ use this name for flags in descriptions as well. The same for structs, unions,
 fields, etc. For syscall variants, use the command name after the `$` sign.
 For example, `fcntl$F_GET_RW_HINT`, `ioctl$FIOCLEX`, `setsockopt$SO_TIMESTAMP`.
 
+If you need to describe several variants of the same kernel struct, the naming
+convention understood by `syz-check` is `<ORIGINAL_KERNEL_NAME>_some_suffix`.
```

Comment:
That's what we use now.
But maybe we should switch to <ORIGINAL_KERNEL_NAME>$some_suffix as we do for syscalls?...

Reply:
If you choose '$', it needs to be emphasized that '$' has no semantic meaning, as otherwise it'll add to confusion.